### DISCUSSION TITLE
How can I contribute to this project?

### DISCUSSION THREAD

**Discussion by DanMarqz** [dnplus/streamlit-oauth#54] [Q&A] [answered]
Hello!

I would like to contribute to this project, at least to the examples part, since with this component I made a functional oauth button with bitbucket and I would like to add it as an example, since I have not seen anywhere on the web how to do it.

Additionally, I am working on making an oauth with Jira and I would also like to be able to add it as an example.

Thank you very much.

**Comment by dnplus** [comment, marked as answer]
Hello @DanMarqz 

You can fork this project & create a pull request to merge your changes
